partner_contact_in_several_companies/tests/test_partner_contact_in_several_companies.py

```diff
@@ -32,7 +34,7 @@ def setUp(self):
                  'partner_contact_in_several_companies',
                  'action_partner_form'),
                 ]:
-            r = ModelData.get_object_reference(cr, uid, module, name)
+            r = ModelData.get_object_reference(module, name)
```

Comment:
I propose to replace the full block by:
```
self.main_partner = self.env.ref('base.main_partner')
self.bob_contact = self.env.ref('partner_contact_in_several_companies.res_partner_contact1')
self.bob_job1 = self.env.ref('partner_contact_in_several_companies.res_partner_contact1_work_position1')
self.roger_contact = self.env.ref('base.res_partner_main2')
self.roger_job2 = self.env.ref('partner_contact_in_several_companies.res_partner_main2_position_consultant')
self.base_partner_action = self.env.ref('base.action_partner_form')
self.custom_partner_action = self.env.ref('partner_contact_in_several_companies.action_partner_form')
```

I find this easier to read, clearer that it defines attributes, and now that you have recordsets, renaming the attributes without the trailing `_id`.